### ISSUE TITLE
Problem with short prefixes

### ISSUE THREAD

**Issue by ZielarSRC** [JeanLucPons/VanitySearch#43]
A message appeared in the forum about a problem that appeared when searching for short prefixes beginning with "3". Here is the original content of the applicant:

> Hi. C:\VanitySearch\VanitySearch.exe -o A:\result.txt -nosse -c 3P
3P112wcuMsGWJ1jLAouxu5tDwTzKUJGjyN
3P114dg7NYteQctAmS8cgGt3c3cJxeMJJ6
3P11gVc4n7wVizvku88ic1yydNwa5GY5ae
and etc
Why are addresses not randomly generated?
...next try:
C:\VanitySearch\VanitySearch.exe -o A:\result.txt -nosse 3E
3E11oBnqGw3cwkk6U4ekid3xm6LjA8F3mm
3E11UemvQxkVTY6KMuqmwfb2utfi3i2rFB
3E11PHRhUXNiqcyuMzEo8kTNFJNNDbrmBn
3E11nkf3TXRKMeHPRMS61yKECaYREFB2uD
random goes after 4 characters

To sum up: the user wanting to find two-character prefixes - receives the results consisting of these two characters and additionally two ones (only). No other prefixes are caught (e.g. 3PP).

The problem is small, but if it occurs it can invisibly affect the results in larger searches somewhere - hence please check this problem in the code.

**Comment by ratijas** [comment]
I guess, that's not the problem with the code. That's just the way brute force works on elliptic curve points.

You're free to generate thousands of addresses and then choose one to your liking. Or specify particular longer prefix without 11.

**Comment by sergeyvb** [comment]
Same problem with prefix 1. if the prefix is ​​short up to 4 characters, it searches for non-random addresses. Example: prefix 12, starts looking for 1211 and so always. How can this be fixed? Works with this correctly https://github.com/samr7/vanitygen
